cluster-autoscaler/processors/status/eventing_scale_up_processor.go

```diff
@@ -46,3 +47,22 @@ func (p *EventingScaleUpStatusProcessor) Process(context *context.AutoscalingCon
 // CleanUp cleans up the processor's internal structures.
 func (p *EventingScaleUpStatusProcessor) CleanUp() {
 }
+
+func reasonsMessage(noScaleUpInfo NoScaleUpInfo) string {
```

Comment:
This looks useful for other implementations of processor. Can you make it public?